meteor/client/ui/DndListWrapper.tsx

```diff
@@ -0,0 +1,66 @@
+import { DropTargetMonitor, useDrop } from 'react-dnd'
+import { DragDropItemTypes } from './DragDropItemTypes'
+import React, { useEffect, useState } from 'react'
+import update from 'immutability-helper'
+import { DndListItemWrapper } from './DndListItemWrapper'
+
+interface IDndListWrapperProps {
+	className: string
+	dndType: DragDropItemTypes
+	stateUIArray: any[]
```

Comment:
`stateUIArray` is a weird name.
Your component is called `DndListWrapper`, why not call this `list`? Or if you want to be really explicit: `listToWrap`

Reply:
Changed!